port/win32/omrfiletext.c

```diff
@@ -192,31 +191,26 @@ int32_t
 omrfile_get_text_encoding(struct OMRPortLibrary *portLibrary, char *charsetName, uintptr_t nbytes)
 {
 	CPINFOEX cpix;
-	BOOL rc;
-	char *c_ptr;
 
-	if (charsetName == NULL) {
+	if (NULL == charsetName) {
 		return -1;
-	}
-
-	rc = GetCPInfoEx(GetConsoleOutputCP(), 0, &cpix);
-	if (rc == FALSE) {
+	} else if (FALSE == GetCPInfoEx(GetConsoleOutputCP(), 0, &cpix)) {
```

Comment:
- `GetConsoleOutputCP` can fail. Should its return value should be verified before passing it to `GetCPInfoEx`?
- Also, `GetCPInfoEx` returns a `bool`. So, we can use `!GetCPInfoEx` instead of `FALSE == !GetCPInfoEx`?
```suggestion
	} else if (!GetCPInfoEx(GetConsoleOutputCP(), 0, &cpix)) {
```

Reply:
Fixed in 901ae9e35bb330b5ce4fc1f5d0ea8f4b7ded11ae.

I expect that if `GetConsoleOutputCP()` failed (returning 0) then `GetCPInfoEx()` would fail due to invalid input, but I the check explicit.

`GetCPInfoEx()` returns `BOOL`, not `bool`, but your suggestion of using `!` is still reasonable.